packages/noir-ethereum-api/src/noir/oracles/accountOracles.ts

```diff
@@ -36,7 +41,33 @@ export const getAccountOracle = async (client: PublicClient, args: string[][]):
 
   return [
     encodeField(account.balance),
-    account.codeHash,
+    encodeHex(account.codeHash),
     encodeField(account.nonce),
   ];
 }
+
+export function encodeAccount(ethProof: GetProofReturnType): AccountWithProof {
+  return {
+    balance: "",
```

Comment:
```ts
balance: encodeField(ethProof.balance)
```

Reply:
Yes working on that, there is a problem with BigNumber to bigint conversion. Added TODO comment